ArchiSteamFarm/Steam/Interaction/Commands.cs

```diff
@@ -2411,6 +2415,50 @@ public sealed class Commands {
 			return responses.Count > 0 ? string.Join(Environment.NewLine, responses) : null;
 		}
 
+		private async Task<string?> ResponsePointsBalance(ulong steamID) {
+			if ((steamID == 0) || !new SteamID(steamID).IsIndividualAccount) {
+				throw new ArgumentOutOfRangeException(nameof(steamID));
+			}
+
+			if (!Bot.HasAccess(steamID, BotConfig.EAccess.Master)) {
+				return null;
+			}
+
+			if (!Bot.IsConnectedAndLoggedOn) {
+				return FormatBotResponse(Strings.BotNotConnected);
+			}
+
+			int? points = await Bot.ArchiWebHandler.GetPointsBalance().ConfigureAwait(false);
+
+			if (points == null) {
+				return FormatBotResponse(string.Format(CultureInfo.CurrentCulture, Strings.WarningFailed));
```

Comment:
Remove call to `string.Format()`, you're not formatting anything here, `Strings.WarningFailed` is enough.